plotpal/volumes.py

```diff
@@ -656,7 +678,7 @@ def plot_colormesh(self, dsets, ni, pl, **kwargs):
         else:
             self.vmin[0], self.vmax[0] = self._get_minmax(self.eq_data['field'])
         cmap = matplotlib.cm.get_cmap(self.cmap)
-        self.data_dicts = [self.out_data, self.mer_data, self.eq_data]
+        self.data_dicts = [self.out_data, self.in_data, self.mer_data, self.eq_data]
```

Comment:
put this in an if statement that only includes in_data if the inner radius is > 0